src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails.jsx

```diff
@@ -215,17 +218,49 @@ class GroupEventDetails extends React.Component {
   }
 }
 
-export {GroupEventDetails};
+class GroupEventDetailsEnvWrapper extends React.Component {
```

Comment:
Initially copied the original design with a components that wraps `GroupEventDetails`, but perhaps the organization environment logic should just exist inside `GroupEventDetails`?